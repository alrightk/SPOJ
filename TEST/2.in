53
23
51
634
34
63
412412
213
42
124
1
