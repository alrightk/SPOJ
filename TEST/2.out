53
23
51
634
34
63
412412
213
